### PR TITLE
Add multichallenge

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
Please merge the #8 first

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # shibboleth-plugin
 
+### Min. shibboleth-idp version:
+4.3.x
+
 ### Installation:
 1. **Copy the package to the server:**
     - Go to: `/privacyIDEA-dist/target/`.
```
Comment (nilsbehlen) [inline README.md:8]: This has to be adjusted if the targets folder are not checked in anymore

File: privacyIDEA-api/target/idp-plugin-privacyIDEA-api-0.0.1.pom.asc [hunk 2]
```diff
@@ -1,16 +1,16 @@
 -----BEGIN PGP SIGNATURE-----
 
-iQIzBAABCAAdFiEE9chR/w1w5heEDr5rB2a5Hkr+j4sFAmTLevsACgkQB2a5Hkr+
```
Comment (nilsbehlen) [inline privacyIDEA-api/target/idp-plugin-privacyIDEA-api-0.0.1.pom.asc:3]: The files in target have been added to gitignore, so they should be removed

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

File: privacyIDEA-api/target/idp-plugin-privacyIDEA-api-0.0.1.pom.asc (hunk 2 above)
Comment (lukasmatusiewicz) [reply privacyIDEA-api/target/idp-plugin-privacyIDEA-api-0.0.1.pom.asc:3]: [21ed4f0](https://github.com/privacyidea/shibboleth-plugin/pull/7/commits/21ed4f0f925a81580caa53da545f32d1a9ced0ad)

File: README.md (hunk 1 above)
Comment (lukasmatusiewicz) [reply README.md]: [9ce729e](https://github.com/privacyidea/shibboleth-plugin/pull/7/commits/9ce729e9bf1ece1d87cd5b6a1b7e5a024468e482)

**Comment by lukasmatusiewicz** [comment]
ready to merge

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**